SpecCheck.py

```diff
@@ -91,6 +88,8 @@ def re_tag_compile(tag):
 obsoletes_regex = re.compile(r'^Obsoletes:\s*(.*)', re.IGNORECASE)
 conflicts_regex = re.compile(r'^(?:Build)?Conflicts:\s*(.*)', re.IGNORECASE)
 
+lconfig_regex = re.compile(r'%{?run_ldconfig}?', re.IGNORECASE)
```

Comment:
What the heck is `%run_ldconfig`? Is it similar to this? https://fedoraproject.org/wiki/Changes/Removing_ldconfig_scriptlets#Upgrade.2Fcompatibility_impact

Reply:
Nope that is some freaky old macro that is direct translation to /sbin/ldconfig